core/src/components/input/input.ionic.scss

```diff
@@ -15,6 +18,16 @@
   min-height: 48px;
 }
 
+// Input Bottom Content
+// ----------------------------------------------------------------
+
+.input-bottom {
+  @include padding-horizontal(0, 0);
+
+  // TODO(FW-6112): Verify the design token is correct once it's available and remove the hardcoded value.
+  font-weight: var(--ionic-font-medium, 500);
+}
```

Comment:
While I was implementing the valid state, I noticed that some styles were missing from the helper text.